src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbol.cs

```diff
@@ -264,15 +267,15 @@ private static void GetAccessorDeclarations(
                             throw ExceptionUtilities.UnexpectedValue(accessor.Kind());
                     }
 
-                    usesFieldKeyword = usesFieldKeyword || containsFieldKeywordInAccessor(accessor);
+                    usesFieldKeyword = usesFieldKeyword || containsFieldExpressionInAccessor(accessor);
                 }
             }
             else
             {
                 var body = GetArrowExpression(syntax);
                 hasGetAccessorImplementation = body is object;
                 hasSetAccessorImplementation = false;
-                usesFieldKeyword = body is { } && containsFieldKeywordInGreenNode(body.Green);
+                usesFieldKeyword = body is { } && containsFieldExpressionInGreenNode(body.Green);
```

Comment:
these feel like they could be pulled up higher.  we just need to walk the entire PropertyDecl looking for FieldExpression.  i don't see the need for breaking this into pieces.

Reply:
With https://github.com/dotnet/roslyn/pull/75325, we're tracking the state for `get` and `set` separately.

Reply:
got it.  thanks :)